Yank/pipeline.py

```diff
@@ -1347,7 +1348,7 @@ def _setup_system(self, system_file_path, pack, alchemical_charge,
                 for i, mol_id in enumerate(molecule_ids):
                     if mol_id not in self._pos_cache:
                         self._pos_cache[mol_id] = utils.get_oe_mol_positions(
-                                utils.read_oe_molecule(self.molecules[mol_id]['filepath']))
+                                utils.load_oe_molecules(self.molecules[mol_id]['filepath'], molecule_idx=0))
```

Comment:
Do you mean to load the first index on every iteration? If I read this right, its loading multiple files, where we assume each has a single molecule in it (or at least is the first molecule). But if I'm wrong, then this behavior I don't think is right.

Brings the other question: how do we handle when the user has a mol2 file with multiple molecules they want to use?